README.md

```diff
@@ -0,0 +1,25 @@
+# The Dollar Effect
+
+A subscription system to sponsor a non-profit organization of your choice for $1 a month.
+
+## Getting Started
+
+To be able to make changes in the repository, you need to be added to the github of dollareffect.github.io.
+Right now, only Albert can add members. Tag him in the project text channel so he can add you.
+
+### Prerequisites
+
+1. Text Editor ([Visual Studio Code](https://code.visualstudio.com), [Sublime Text](https://www.sublimetext.com))
+2. Netlify CLI (https://www.netlify.com/products/cli/)
+
+### Local Development
+
+In the project root directory terminal, run 'netlify dev' to start local testing of your changes. The browser should open and you should be able to access it through 'http://localhost:8888'.
+
+```
+netlify dev
+```
```

Comment:
Is netlify really required?

Reply:
Netlify is NOT really required to open/run the html file. However, during testing my changes opening the html file, some icons and images were not shown. It seemed like the html file was not able to refer to some icons and images even though it is there in the files. Opening/Debugging through netlify allows these kind of problems to be solved - additionally running it through netlify is a breeze since it is beginner-friendly. Let me know what you think of my explanation. Thank you!

Reply:
I get it but I don't get how did we come uo with Netlify wherein we host this website in GitHub. Can we follow the guide to use [Jekyll ](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll)instead to locally serve this site?

Reply:
I will remove netlify in the pre-requisite of this readme and then add both Jekyll and Netlify as a local development options for the dev. This documentation for Jekyll is really useful and I have only known and tried Netlify that time. Thank you for this review!